tests/MongoDB.Driver.Tests/Specifications/retryable-writes/RetryableWriteTestRunner.cs

```diff
@@ -55,6 +55,11 @@ private void VerifyServerRequirements(BsonDocument definition)
         {
             RequireServer.Check().ClusterType(ClusterType.ReplicaSet);
 
+            if (CoreTestConfiguration.GetStorageEngine() == "mmapv1")
+            {
+                throw new SkipException("Test skipped because mmapv1 does not support retryable writes.");
```

Comment:
We're going to need a lot more of these in the later ticket to automate the tests

Reply:
Do the JSON files specify which storage engines tests can be run on? Probably not.

Reply:
You are correct: they do not unfortunately